components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx

```diff
@@ -27,21 +27,21 @@ export default function DistanceTravelledMode({ data }) {
       </Flex>
 
       {/* CONTAINER FOR INDIVIDUAL METHOD AND ACTIVE/PUBLIC/SHARED/METHOD   */}
-      <Flex direction={["column", "row"]} width="100%">
+      <Flex direction={["column", "row"]} width="100%" my="15px">
         <Flex
           direction="column"
           width={["100%", "50%"]}
           align={["left", "center"]}
+          justifyContent="space-between"
         >
           {/* row for the sub title */}
-          <Flex direction="row">
+          <Flex direction="row" alignItems={"center"} my="5px">
```

Comment:
`alignItems` is `align` in Chakra, no need `{` here 

```suggestion
          <Flex direction="row" align="center" my="5px">
```